docs/release-notes/2.41.1.md

```diff
@@ -33,4 +33,6 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 
 ## Bugs Fixed
+- [PMM-9825](https://jira.percona.com/browse/PMM-9825) - [QAN] When PostgreSQL was monitored with [pg_stat_monitor](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#pg_stat_monitor), User ID was displayed instead of real User Names in Query Analytics. This issue was fixed.
```

Comment:
```suggestion
- [PMM-9825](https://jira.percona.com/browse/PMM-9825) - [QAN] When PostgreSQL was monitored with [pg_stat_monitor v2.0+](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/postgresql.html#pg_stat_monitor), User ID was displayed instead of real User Names in Query Analytics. This issue was fixed.
```